XMonad/Actions/BindByLayout.hs

```diff
@@ -27,23 +27,25 @@ import XMonad.StackSet as S
 --
 -- You can use this module with the following in your @~\/.xmonad\/xmonad.hs@:
 --
--- >  import XMonad.Actions.PerLayoutKeys
+-- >  import XMonad.Actions.BindByLayout
 --
--- >   ,((0, xK_F2), bindByLayout [("Tall", spawn "rxvt"), ("Mirror Tall", spawn "xeyes"), ("", spawn "xmessage hello")])
+-- >   ,((0, xK_F2), bindBL [("Tall", spawn "rxvt"), ("Mirror Tall", spawn "xeyes"), ("", spawn "xmessage hello")])
 --
 -- For detailed instructions on editing your key bindings, see
 -- "XMonad.Doc.Extending#Editing_key_bindings".
 
 -- | Uses supplied function to decide which action to run depending on current layout name.
-chooseActionByLayout :: (String->X()) -> X()
-chooseActionByLayout f = withWindowSet (f . description . S.layout. S.workspace . S.current)
+chooseAction :: (String->X()) -> X()
+chooseAction f = withWindowSet (f . description . S.layout. S.workspace . S.current)
```

Comment:
[chooseAction](https://hackage.haskell.org/package/xmonad-contrib-0.17.0/docs/XMonad-Actions-PerWorkspaceKeys.html#v:chooseAction) already exists, which is probably why this one was chosen

Reply:
Exactly. `chooseAction` is exported by `PerWorkspaceKeys`; rather than require someone using both to import both modules qualified, I renamed this one.